pyvista/core/pointset.py

```diff
@@ -1850,57 +1849,92 @@ class StructuredGrid(_vtk.vtkStructuredGrid, PointGrid, StructuredGridFilters):
 
     Can be initialized in one of the following several ways:
 
-    - Create empty grid
-    - Initialize from a vtk.vtkStructuredGrid object
-    - Initialize directly from the point arrays
+    * Create empty grid
+    * Initialize from a ``vtk.vtkStructuredGrid`` object
+    * Initialize directly from the point arrays
```

Comment:
This "from the point arrays" might be misleading, because every mesh has an `(n, 3)`-shaped point array, and instead here we have three `(nx, ny, nz)`-shaped arrays defining the grid through the x, y and z coordinates separately.

Reply:
Resolved in 01572dd3332b8be3937903e38fcfd36df85496b1.